LGTM, would you mind fixing the linting issue
```
pythonforandroid/recipes/android/src/android/permissions.py:452:32: E261 at least two spaces before inline comment
```

--- a/pythonforandroid/recipes/android/src/android/permissions.py
+++ b/pythonforandroid/recipes/android/src/android/permissions.py
@@ -449,7 +449,7 @@ class Permission:
     WRITE_VOICEMAIL = (
         "com.android.voicemail.permission.WRITE_VOICEMAIL"
         )
-    MANAGE_EXTERNAL_STORAGE = ( # Convenient use of paths to manage files
+    MANAGE_EXTERNAL_STORAGE = (  # Convenient use of paths to manage files
         "android.permission.MANAGE_EXTERNAL_STORAGE"
         )
 

